src/app/modules/admin-core/guards/route.guard.ts

```diff
@@ -9,20 +9,27 @@ import { UsersService } from '../services/users-service/users.service';
   providedIn: 'root'
 })
 export class RouteGuard implements CanActivate {
-
+  promiseRowData: any;
+  rolesArray: any;
   constructor(private usersService: UsersService, private route: Router) {
   }
   canActivate(
     next: ActivatedRouteSnapshot,
     state: RouterStateSnapshot): Observable<boolean> | Promise<boolean> | boolean {
-   
+    this.promiseRowData = this.usersService.getUserRoles();
+    if (this.promiseRowData['result']) {
+      this.rolesArray = this.promiseRowData['result'].roles;
+    }
     if (localStorage.getItem('access-token')) {
```

Comment:
Why is this check here? Currently, we are not saving anything in local storage

Reply:
Token we are storing